media-tv/gentoo-vdr-scripts/gentoo-vdr-scripts-3.0_rc2-r1.ebuild

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit tmpfiles user-info
+
+DESCRIPTION="Scripts necessary for use of VDR as a set-top-box"
+HOMEPAGE="https://gitweb.gentoo.org/proj/gentoo-vdr-scripts.git/about/"
+SRC_URI="https://github.com/madmartin/gentoo-vdr-scripts/archive/refs/tags/${P}.tar.gz"
+S="${WORKDIR}/${PN}-${P}"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~x86"
+
+DEPEND="
+	acct-group/vdr
+	acct-user/vdr"
+RDEPEND="${DEPEND}
+	app-admin/sudo
+	sys-process/wait_on_pid"
+BDEPEND="${DEPEND}"
+
+src_install() {
+	default
+
+	VDR_HOME="$(egethome vdr)"
+	echo "VDR_HOME=${VDR_HOME}"
+	diropts -ovdr -gvdr
+	keepdir "${VDR_HOME}"
```

Comment:
Is this actually needed? HOME is created by acct-*?

Reply:
Yes, you are right... this is double stitched.
acct-user/vdr does a `keepdir` first, so we really dont need it here. I will fix that.